ui/src/components/PollBody/PollBody.jsx

```diff
@@ -0,0 +1,338 @@
+import React, { useState, useEffect } from 'react';
+import AutoSuggest from 'react-autosuggest';
+import Voter from '../Voter/Voter';
+import AnswerOption from '../AnswerOption/AnswerOption';
+import API from '../../API';
+import members from '../../data/voters.json';
+import './PollBody.css';
+
+const PollBody = () => {
+    const [currPage, setCurrPage] = useState(0);
+    const [privacy, setPrivacy] = useState('private');
+    const [value, setValue] = useState('');
+    const [suggestions, setSuggestions] = useState([]);
+
+    const [pollTitle, setPollTitle] = useState("");
+    const [pollDescription, setPollDescription] = useState("");
+    const [pollType, setPollType] = useState("");
+    const [pollExpiration, setPollExpiration] = useState();
+    const [voters, setVoters] = useState([]);
+    const [optionInds, setOptionInds] = useState([1, 2]);
+    const [numForceUpdates, setNumForceUpdates] = useState(0); // BAD PRACTICE ;-;
+    const [options, setOptions] = useState([{optionName: "", description:""}, {optionName:"", description:""}]);
```

Comment:
If your components here had their props change occasionally (i.e. be on multiple levels),
React would automatically re-render functional components, should their properties change,
which isn't happening here becaue they're all in the same component.